eslint.config.cjs

```diff
@@ -0,0 +1,167 @@
+const typescriptEslint = require('@typescript-eslint/eslint-plugin')
+const jest = require('eslint-plugin-jest')
+const simpleImportSort = require('eslint-plugin-simple-import-sort')
+const _import = require('eslint-plugin-import')
+const localRules = require('eslint-plugin-local-rules')
+const path = require('path')
+
+const { fixupPluginRules } = require('@eslint/compat')
+
+const globals = require('globals')
+const tsParser = require('@typescript-eslint/parser')
+const js = require('@eslint/js')
+
+const { FlatCompat } = require('@eslint/eslintrc')
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+})
+
+const project = path.resolve(__dirname, 'tsconfig.json')
+
+module.exports = [
+  {
+    files: ['**/*.ts'],
+  },
```

Comment:
With the v9 release eslint give's a way to define rules for specific file extensions. in this block you have only a file extensions without inserting rules and others things. Are that really working good?

Reply:
Yes. This is telling ESLint to search for `**/*.ts` files by default. This replaces the `--ext .ts` command line flag. You can optionally include additional configuration if you want, but the primary focus of this entry is to ensure that TypeScript files are search for by default.